manifests/r/RileyTestut/AltServer/1.6.00/RileyTestut.AltServer.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=QUSU.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RileyTestut.AltServer
+PackageVersion: 1.6.00
+InstallerType: msi
+Scope: machine
+UpgradeBehavior: install
+ReleaseDate: 2023-01-30
+AppsAndFeaturesEntries:
+- ProductCode: '{EE0087EA-3FB6-4AE1-9522-7BEB70A038EE}'
+  UpgradeCode: '{7D1A38CC-208D-4D3C-8D8B-BDFEC732880A}'
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Apple.iTunes
Installers:
```

Reply:
The package installs, but requires iTunes to run: 
![image](https://user-images.githubusercontent.com/114952565/218160401-1fb15ef2-cf9b-48ed-aade-549039d9f925.png)